app/models/pipeline_run.rb

```diff
@@ -19,6 +19,7 @@ class PipelineRun < ApplicationRecord
 
   DEFAULT_SUBSAMPLING = 1_000_000 # number of reads to subsample to, after host filtering
   OUTPUT_JSON_NAME = 'idseq_web_sample.json'.freeze
+  MULTIHIT_OUTPUT_JSON_NAME = 'multihit_idseq_web_sample.json'.freeze
```

Comment:
Seems unwise to rename this file.

Reply:
But if it has to be renamed, the decision on which filename to use should be made based on pipeline version not pipeline branch.

Reply:
An alternative way of implementing such a rename without complicating the code with version checks is as follows

1)  Make pipeline emit both files, compatible with old webapp

2) In a week or two, make webapp only look for the new file, and make pipeline no longer generate the old one.

We should also get rid of that "wait for all files to be generated with timestamps after the job submission" approach, as it leads to cumbersome situations like this one.

Reply:
I agree, now that we have versioned results folders, we could have kept the name the same. The name change was meant to avoid any confusion because the meaning of what's in the file changes drastically from the old pipeline version to the new one.